packages/ag-charts-community/src/chart/agChartV2.ts

```diff
@@ -509,7 +509,8 @@ function applySeriesOptionModules(series: Series<any>, options: AgBaseSeriesOpti
     const seriesOptionModules = REGISTERED_MODULES.filter((m): m is SeriesOptionModule => m.type === 'series-option');
 
     for (const mod of seriesOptionModules) {
-        if (mod.optionsKey in options) {
+        const supportedSeriesTypes: readonly string[] = mod.seriesTypes;
```

Comment:
Nit: Is the `: readonly string[]` needed? I would have thought the infered type from `mod.seriesTypes` should already be `string[]` or something stricter?

Reply:
That's what I would have thought, but using `mod.seriesTypes` directly causes this compiler error:
```
packages/ag-charts-community/src/chart/agChartV2.ts:513:67 - error TS2345: Argument of type 'string' is not assignable to parameter of type '"line" | "scatter" | "bubble" | "area" | "bar" | "box-plot" | "histogram" | "heatmap" | "waterfall" | "range-bar" | "range-area" | "pie" | "radar-line" | "radar-area" | "radial-bar" | "radial-column" | "nightingale" | "treemap"'.

513         if (mod.optionsKey in options && mod.seriesTypes.includes(series.type)) {
                                                                      ~~~~~~~~~~~
```